### PR TITLE
change usage of user_id via session

### PR DESCRIPTION
**Pull request by andromash**

### CODE REVIEW

File: src/main/java/com/internet/shop/controllers/shoppingcart/DeleteProductFromTheCartController.java [hunk 1]
```diff
@@ -23,8 +23,9 @@
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
         Long productId = Long.valueOf(req.getParameter("id"));
+        Long userId = (Long) req.getSession().getAttribute(USER_ID);
```
Comment (AndrewPolishchuc) [inline src/main/java/com/internet/shop/controllers/shoppingcart/DeleteProductFromTheCartController.java]: You can use the already defined constant(USER_ID) of the "AddToCartController" class so as not to create a new constant in every controller

File: src/main/java/com/internet/shop/controllers/shoppingcart/GetCurrentCartController.java [hunk 2]
```diff
@@ -14,18 +14,19 @@
 
 @WebServlet("/cart/products")
 public class GetCurrentCartController extends HttpServlet {
-    private static final long USER_ID = 1;
+    private static final String USER_ID = "user_id";
```
Comment (AndrewPolishchuc) [inline src/main/java/com/internet/shop/controllers/shoppingcart/GetCurrentCartController.java]: Same

File: src/main/java/com/internet/shop/controllers/user/GetUsersOrdersController.java [hunk 3]
```diff
@@ -13,15 +13,16 @@
 
 @WebServlet("/user/orders")
 public class GetUsersOrdersController extends HttpServlet {
-    private static final long USER_ID = 1;
+    private static final String USER_ID = "user_id";
```
Comment (AndrewPolishchuc) [inline src/main/java/com/internet/shop/controllers/user/GetUsersOrdersController.java]: Same

**Review by AndrewPolishchuc — APPROVED** [review]
Good job. I think that you need to remove unnecessary definition of constants and use a constant(USER_ID) of one class but in general it is in your personal opinion.

File: src/main/java/com/internet/shop/controllers/shoppingcart/DeleteProductFromTheCartController.java (hunk 1 above)
Comment (andromash) [reply src/main/java/com/internet/shop/controllers/shoppingcart/DeleteProductFromTheCartController.java]: agree, genius

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**